src/demo/quality-levels.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+
+<head></head>
+
+<body>
+  <h1>Mux Video.js Kit</h1>
+
+  <section>
+    <h2>Quality Levels example</h2>
+    <video id="mux-default" class="video-js vjs-16-9" controls preload="auto" width="100%"
+      poster="https://image.mux.com/DS00Spx1CV902MCtPj5WknGlR102V5HFkDe/thumbnail.jpg"
+      data-setup='{
+        "plugins": {
+          "httpSourceSelector": {}
+        }
```

Comment:
does this plugin value: `httpSourceSelector` activate the `videojs-contrib-quality-levels` functionality?

I see we're including `videojs-contrib-quality-levels` in webpack.dev, so that must be loaded in all the demos, right?

Reply:
It activates `videojs-http-source-selector` specifically, which internally activates `videojs-contrib-quality-levels`.
The plugin is loaded into video.js in all dev pages, but doesn't activate in other pages. I wasn't sure exactly how to target a specific demo page in webpack and figured since it's dev and that it only runs if enabled like here, it's fine if it's included in all the demo pages.

Reply:
Got it, makes sense 👍🏼